CONTRIBUTING.md

```diff
@@ -16,65 +16,71 @@ We want to make contributing to this project as easy and transparent as possible
 ## Flow
 We are using [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow), so all code changes happen through pull requests from a [forked repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-### Versioning
-We use [semver](https://semver.org/) naming convention.
-
 ### Branching
 - The current active branch is `next`. Every branch with a fix/feature must be forked from `next`.
 
-- The branch name should contain a short issue/feature description separated with hyphens [(kebab-case)](https://en.wikipedia.org/wiki/Letter_case#Kebab_case).
+- The branch name should contain a short issue/feature description separated with hyphens [(kebab-case)](https://en.wikipedia.org/wiki/Letter_case#Kebab_case). 
 
-For example, if the issue title is `Fix functionality X in component Y` then the branch name will be something like: `fix-x-in-y`.
+    For example, if the issue title is `Fix functionality X in component Y` then the branch name will be something like: `fix-x-in-y`.
 
 - New branch should be rebased from `next` before submitting a PR in case there have been changes to avoid merge commits.
 i.e. this branches state:
-```
+  ```
```

Comment:
I suggest to align the code block with the bullet, because it is logically at the same layer